docs/sebak_deployment.md

```diff
@@ -0,0 +1,192 @@
+
+After successfully creating genesis block and SSL certificates, run *n0*:
+
+```
+$ sebak node \
+    --network-id "this-is-test-sebak-network" \
+    --bind "https://localhost:12345" \
+    --tls-key "sebak.key" \
+    --tls-cert "sebak.crt" \
+    --storage "file:///tmp/db-n0" \
+    --secret-seed SCN4NSV5SVHIZWUDJFT4Z5FFVHO3TFRTOIBQLHMNPAZJ37K5A2YFSCBM \
+    --validators "https://localhost:12346?address=GDPQ2LBYP3RL3O675H2N5IEYM6PRJNUA5QFMKXIHGTKEB5KS5T3KHFA2 https://localhost:12347?address=GCZG7MBKRSS6MJVZOALYBJB5C223FSZ43MDTPX2O4UGQTCXTHWBDNUB6" \
+    --log-level debug \
+
+INFO[10-31|19:18:35] Starting Sebak                           module=main caller=run.go:401
+DBUG[10-31|19:18:35] parsed flags:                            module=main
+	network-id=this-is-test-sebak-network
+	bind=https://localhost:12345
+	publish=https://localhost:12345
+	storage=file:///tmp/db-n0
+	tls-cert=sebak.crt
+	tls-key=sebak.key
+	log-level=debug
+	log-format=terminal
+	log=
+....
+```
+
+and *n1*
+```
+$ sebak node \
+    --network-id "this-is-test-sebak-network" \
+    --bind "https://localhost:12346" \
+    --tls-key "sebak.key" \
+    --tls-cert "sebak.crt" \
+    --storage "file:///tmp/db-n1" \
+    --secret-seed SBGJDQ2J4PIYU7JVGKIBLNF6X3DOEVW3I4W2T77M2B47X2MPSUNXZ7T7 \
+    --validators "https://localhost:12345?address=GBNUTWSM4FRSEULVMHZF7NFQWIBGEDF5X5OHXFOZJB6SH5MIEDEJEJ2F https://localhost:12347?address=GCZG7MBKRSS6MJVZOALYBJB5C223FSZ43MDTPX2O4UGQTCXTHWBDNUB6" \
+    --log-level debug \
+
+INFO[10-31|19:18:35] Starting Sebak                           module=main caller=run.go:401
+DBUG[10-31|19:18:35] parsed flags:                            module=main
+	network-id=this-is-test-sebak-network
+	bind=https://localhost:12346
+	publish=https://localhost:12346
+	storage=file:///tmp/db-n1
+	tls-cert=sebak.crt
+	tls-key=sebak.key
+	log-level=debug
+	log-format=terminal
+	log=
+....
+```
+
+and *n2*
+```
+$ sebak node \
+    --network-id "this-is-test-sebak-network" \
+    --bind "https://localhost:12347" \
+    --tls-key "sebak.key" \
+    --tls-cert "sebak.crt" \
+    --storage "file:///tmp/db-n2" \
+    --secret-seed SDQKKG2MBSAXVLUE5JFNM7MXQ7MV7WPRIEOS7U7KLWFDKYDKXTLSSRTC \
+    --validators "https://localhost:12345?address=GBNUTWSM4FRSEULVMHZF7NFQWIBGEDF5X5OHXFOZJB6SH5MIEDEJEJ2F https://localhost:12346?address=GDPQ2LBYP3RL3O675H2N5IEYM6PRJNUA5QFMKXIHGTKEB5KS5T3KHFA2" \
+    --log-level debug \
+
+INFO[10-31|19:18:35] Starting Sebak                           module=main caller=run.go:401
+DBUG[10-31|19:18:35] parsed flags:                            module=main
+	network-id=this-is-test-sebak-network
+	bind=https://localhost:12347
+	publish=https://localhost:12347
+	storage=file:///tmp/db-n2
+	tls-cert=sebak.crt
+	tls-key=sebak.key
+	log-level=debug
+	log-format=terminal
+	log=
+....
+```
+
+If nodes launched sucessful, each node will connect other nodes and will be ready to make consensus.
+
+# Spinning Network using Docker
```

Comment:
I figure, using docker scenario is simpler than command line.
How about putting it up? It’s just my personal opinion.

Reply:
I understand. 
But I leave this phrase as it is. 
thanks for advice. :)